src/index.html

```diff
@@ -16,7 +16,16 @@
       href="./styles/index.scss"
```

Comment:
The href attribute should point to a `.css` file, not a `.scss` file. SCSS files are preprocessed to CSS and the resulting CSS file should be linked in the HTML document.